diff --git a/build.sbt b/build.sbt
index 98053c7..ba7185e 100644
--- a/build.sbt
+++ b/build.sbt
@@ -39,8 +39,9 @@ resolvers ++= Seq (
 
 // Provide a managed dependency on X if -DXVersion="" is supplied on the command line.
 val defaultVersions = Map(
-  "chisel3" -> "3.2-SNAPSHOT",
-  "chisel-iotesters" -> "1.3-SNAPSHOT",
+  "chisel3" -> "3.2+",
+  "chisel-iotesters" -> "1.2+",
+  "treadle" -> "1.1+",
   "rocketchip" -> "1.2",
   "vegas" -> "0.3.11",
   "handlebars-scala" -> "2.1.1"
